Library/.rubocop_rules.yml

```diff
@@ -1,75 +1,197 @@
+    - 'Homebrew/cask/lib/hbc/macos.rb'
+    - 'Homebrew/cask/lib/hbc/utils.rb'
+
+Metrics/PerceivedComplexity:
   Enabled: false
 
 # favor parens-less DSL-style arguments
 Lint/AmbiguousOperator:
   Enabled: false
+
 Lint/AmbiguousRegexpLiteral:
   Enabled: false
+
 Lint/AssignmentInCondition:
   Enabled: false
+
+Lint/EndAlignment:
+  AlignWith: variable
+
+# `formula do` uses nested method definitions
+Lint/NestedMethodDefinition:
+  Exclude:
+    - 'Homebrew/test/**/*'
+
 Lint/ParenthesesAsGroupedExpression:
   Enabled: false
 
-# compact style
+Style/Alias:
+  EnforcedStyle: prefer_alias
+
+Style/AlignHash:
+  EnforcedHashRocketStyle: table
+  EnforcedColonStyle: table
+  Exclude:
+    - 'Taps/**/*'
+
+# `system` is a special case and aligns on second argument
+Style/AlignParameters:
+  Enabled: false
+
+Style/BarePercentLiterals:
+  Exclude:
+    - 'Taps/**/*'
+  EnforcedStyle: percent_q
+
+Style/BlockDelimiters:
+  EnforcedStyle: semantic
+  Exclude:
+    - 'Taps/**/*'
+  FunctionalMethods:
+    - expect
+    - find
+    - let
+    - let!
+    - subject
+    - watch
+    - inject
+    - map
+    - map!
+    - collect
+    - collect!
+    - reject
+    - reject!
+    - delete_if
+    - with_object
+    - popen_read
+  ProceduralMethods:
+    - after
+    - at_exit
+    - before
+    - benchmark
+    - bm
+    - bmbm
+    - capture_io
+    - capture_output
+    - capture_subprocess_io
+    - chdir
+    - context
+    - create
+    - define_method
+    - define_singleton_method
+    - fork
+    - measure
+    - new
+    - open
+    - realtime
+    - shutup
+    - tap
+    - each
+    - each_pair
+    - each_with_index
+    - reverse_each
+    - ignore_interrupts
+  IgnoredMethods:
+    - each_with_object
+    - it
+    - its
+    - lambda
+    - proc
+    - formula
+    - mock
+    - devel
+    - stable
+    - head
+    - assert_raises
+    - assert_nothing_raised
+    - resource
+    - with_build_environment
+    - ensure_writable
+    - satisfy
+    - fetch
+    - brew
+    - expand
+    - env
+    - recursive_dependencies
+    - trap
+    - link_dir
+    - with_system_path
+
+Style/CaseIndentation:
+  IndentWhenRelativeTo: end
+
+Style/ClassAndModuleChildren:
+  EnforcedStyle: nested
+
+# percent-x is allowed for multiline
+Style/CommandLiteral:
+  EnforcedStyle: mixed
+
+# our current conditional style is established, clear and
+# requiring users to change that now would be confusing.
+Style/ConditionalAssignment:
+  Enabled: false
+
+Style/Documentation:
+ Enabled: false
+
 Style/EmptyLineBetweenDefs:
   AllowAdjacentOneLineDefs: true
 
+# dashes in filenames are typical
+Style/FileName:
+  Regex: !ruby/regexp /^[\w\@\-\+\.]+(\.rb)?$/
+
+# counterproductive in formulas, notably within the install method
+Style/GuardClause:
+  Exclude:
+    - 'Taps/**/*'
+
+# depends_on foo: :bar looks rubbish
+Style/HashSyntax:
+  EnforcedStyle: ruby19_no_mixed_keys
+  Exclude:
+    - 'Taps/**/*'
+
+Style/IfUnlessModifier:
```

Comment:
I left this as-is, because disabling doesn't change anything. Corrected two lines manually though which were > 80 characters, which will be caught automatically once this is fixed.